meteor/client/ui/TestTools/TimelineDatastore.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import {
+	Translated,
+	translateWithTracker,
+	useSubscription,
+	useTracker,
+} from '../../lib/ReactMeteorData/react-meteor-data'
+import { MeteorReactComponent } from '../../lib/MeteorReactComponent'
+import { StudioSelect } from './StudioSelect'
+import { StudioId } from '../../../lib/collections/Studios'
+import { Mongo } from 'meteor/mongo'
+import { TimelineDatastoreEntry } from '../../../lib/collections/TimelineDatastore'
+import { PubSub } from '../../../lib/api/pubsub'
+import { unprotectString } from '@sofie-automation/corelib/dist/protectedString'
+
+const TimelineDatastore = new Mongo.Collection<TimelineDatastoreEntry>('timelineDatastore')
+
+interface ITimelineDatastoreViewProps {
+	match?: {
+		params?: {
+			studioId: StudioId
+		}
+	}
+}
+interface ITimelineDatastoreViewState {}
+const TimelineDatastoreView = translateWithTracker<ITimelineDatastoreViewProps, ITimelineDatastoreViewState, {}>(
```

Comment:
Is there a reason some of these components are classes and some are functions?
Not a big deal, but would be nice for them to be consistent :)